content/en/tracing/trace_explorer/visualize.md

```diff
@@ -0,0 +1,92 @@
+
+{{< img src="tracing/trace_explorer/visualize/list_view_table_controls.mp4" alt="Configuring the display table" video=true style="width:80%;">}}
+
+{{< site-region region="us,eu,us3,us5,gov" >}}
+The default sort for spans in the list visualization is by timestamp, with the most recent spans on top. To surface spans with lowest or highest value for a measure first, or to sort your spans lexicographically for the value of a tag, specify that column as the **by** column.
+
+{{< /site-region >}}
+
+The configuration of the columns is stored alongside other elements of your troubleshooting context in saved views.
+
+## Timeseries
+
+Use timeseries to visualize the evolution of a [measure][3] (or a count of unique tag values) over a selected time frame, and optionally split the data by up to three tags (grouping).
+
+**Note**: The [Live Explorer][4] (15 minutes) allows grouping by only one dimension.
+
+Aggregated views use additional query options, to define the **measured tag dimension**, the dimensions to **group** the query by, and the **aggregation period**. For example:
+
+1. Choose to view the `Duration` measure.
+
+   {{< img src="tracing/trace_explorer/visualize/group_by_measured_dimension.png" alt="Measured Dimension" style="width:100%;">}}
+
+2. Select the aggregation function for the `Duration` measure. Selecting a measure lets you choose the aggregation function whereas selecting a qualitative attribute displays the unique count.
+
+   {{< img src="tracing/trace_explorer/visualize/group_by_aggregation_function.png" alt="Aggregation Function" style="width:100%;">}}
+
+3. Group the query by a dimension, for example, `Resource`.
+
+   {{< img src="tracing/trace_explorer/visualize/group_by_dimension.png" alt="Split Dimension" style="width:100%;">}}
+
+4. Choose to display a number of either top or bottom values according to the selected tag.
+
+    {{< img src="tracing/trace_explorer/visualize/group_by_top_bottom.png" alt="Top Bottom X values" style="width:100%;">}}
+
+5. Choose the rollup period, for example, `10min`.
+
+    {{< img src="tracing/trace_explorer/visualize/group_by_rollup_period.png" alt="Rollup Period" style="width:100%;">}}
+
+The following Trace Explorer timeseries view shows the evolution of the top ten resource names of the service `shopist-web-ui` according to the 95th percentile of `Duration` over the past four hours:
+
+{{< img src="tracing/trace_explorer/visualize/timeseries_view.png" alt="Timeseries view" style="width:100%;">}}
+
+Choose additional display options for timeseries: the **roll-up interval**, whether you **display** results as **bars** (recommended for counts and unique counts), **lines** (recommended for statistical aggregations) or **areas**, and the **colorset**.
+
+## Top list
+
+Use a top list to visualize a span count, a count of unique tag values, or a measure split by one tag dimension.
```

Comment:
is "a count of unique tag values" correct? The previous wording didn't make sense to me. If this is still off, let's workshop what it should say.

Reply:
This makes sense, thanks for modifying